.changeset/slow-bikes-type.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
Thanks for the contribution! Could you follow the [guidelines here](https://github.com/launchdarkly/launchpad-ui/blob/main/CONTRIBUTING.md#step-3-generate-a-changelog-for-your-code) to ensure the changelog entry properly communicates the change to consumers?